migrations/V163__Add_Category_Specific_Columns_To_TRB_Admin_Notes.sql

```diff
@@ -0,0 +1,67 @@
+ALTER TABLE trb_admin_notes
+  -- columns for Initial Request Form category
+  ADD COLUMN applies_to_basic_request_details BOOLEAN,
+  ADD COLUMN applies_to_subject_areas BOOLEAN,
+  ADD COLUMN applies_to_attendees BOOLEAN,
+
+  -- columns for Advice Letter category
+  ADD COLUMN applies_to_meeting_summary BOOLEAN,
+  ADD COLUMN applies_to_next_steps BOOLEAN;
+
+-- populate columns for existing admin notes
+UPDATE trb_admin_notes
+SET
+  applies_to_basic_request_details = FALSE,
+  applies_to_subject_areas = FALSE,
+  applies_to_attendees = FALSE
+WHERE category = 'INITIAL_REQUEST_FORM';
+
+UPDATE trb_admin_notes
+SET
+  applies_to_meeting_summary = FALSE,
+  applies_to_next_steps = FALSE
+WHERE category = 'ADVICE_LETTER';
+
+-- add constraints so that only notes of the correct category can set category-specific columns
```

Comment:
To test that these constraints hold with actual data, I generated and exported some data with `scripts/dev db:seed`, it's saved (on a different branch) at https://github.com/CMSgov/easi-app/blob/EASI-3362/sql-constraints-with-manual-test-data/migrations/V166__Add_Seed_Data_For_Testing.sql. If you want to test yourself, open the [URL with the raw file contents](https://raw.githubusercontent.com/CMSgov/easi-app/EASI-3362/sql-constraints-with-manual-test-data/migrations/V166__Add_Seed_Data_For_Testing.sql), copy-paste it all into this file right before this line (after the two `UPDATE` statements, before adding the constraints), then run `scripts/dev up:backend`. The migration should complete successfully; all the inserted data is valid under the added constraints.